### ISSUE TITLE
color contrast of some text in Gitter UI is poor

### ISSUE THREAD

**Issue by stevefaulkner** [gitterHQ/gitter#771]
the grey and light grey text is difficult to read.

![capture](https://cloud.githubusercontent.com/assets/835859/8703538/8deb85ac-2b15-11e5-836a-ccdb7e1ee251.PNG)
Foreground:#B6AAAA
Background:#FFFFFF
The contrast ratio is: 2.3:1
Text failed at Level AA

as in fails WCAG 2.0 http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

**Comment by mydigitalself** [comment]
Have increased the contrast of the /me message a little, it's being deployed to Next (https://gitter.zendesk.com/hc/en-us/articles/203904511-Gitter-Next) right now.

Also adjusted the timestamp contrast a bit too, although it's intentionally more subtle than the status message to avoid clutter.

**Comment by MichielBijl** [comment]
The contrast ratio on Next, while better, is still poor: 3.54:1

Wouldn't it be better to use a shade of Gitter pink, like #e60550; which passes WCAG Level AA.

![screen shot 2015-08-07 at 17 41 10](https://cloud.githubusercontent.com/assets/5457769/9139608/8fae2a20-3d2b-11e5-9ec4-0871ebc265ef.png)

You see this kind of colour usage in otter chat apps, too. Like Textual 5.

**Comment by mydigitalself** [comment]
Trying our purple out, see how people feel about it :)

**Comment by MichielBijl** [comment]
Cool. You can check the contrast for it on [WebAim's Colour Contrast Checker](http://webaim.org/resources/contrastchecker/).

**Comment by MichielBijl** [comment]
The colour currently in use on Next—colour code: #7936b5—passes WCAG Leve AA, very nice :) It now has a contrast ratio of 6.98:1!
